packages/ckeditor5-watchdog/src/editorwatchdog.ts

```diff
@@ -381,6 +424,7 @@ class EditorWatchdogInitPlugin {
 			evt.stop();
 
 			this.editor.model.enqueueChange( { isUndoable: true }, writer => {
+				this._restoreCollaborationData();
```

Comment:
Not sure if that makes any difference, but in guides we put initialization straight in `init()` of integration plugins, instead of in `'init'` event. Same when it comes to the RTC adapters (they connect before `'init'` event).

Reply:
Maybe there's one difference -- handling RTC, actually. If we would want to prevent using watchdog restore in RTC, it is easier to put `this._restoreCollaborationData()` in the `'init'` callback as it will simply won't get fired.